shells/pipes-shell/source/verbs/run-arc.js

```diff
@@ -18,7 +18,7 @@ const {log, warn} = logsFactory('pipe');
 
 // The implementation was forked from verbs/spawn.js
 export const runArc = async (msg, tid, bus, runtime, env) => {
-  const {recipe, arcid, storageKeyPrefix, pecid} = msg;
+  const {recipe, arcid, storageKeyPrefix, pecid, particleid, particlename} = msg;
```

Comment:
Nit: I would think these would usually be capitalised like so: particleId, particleName.

Same for arcId, pecId for that matter.